.azure-pipelines/tox.yml

```diff
@@ -4,36 +4,36 @@ parameters:
 
 
 jobs:
-- template: run-tox-env.yml@tox
-  parameters:
-    jobs:
-      docs: null
-      lint:
-        before:
-        - task: CacheBeta@0
-          displayName: cache pre-commit
+  - template: run-tox-env.yml@tox
```

Comment:
Sequences shouldn't be indented.

Reply:
For Ansible code in particular that is a serious issue, especially with blocks where it makes extremely hard to find block/rescue/always sections. 

Also non identing makes is much easier to misplace a when condition between block level or last task within the block.

For these reasons all the Ansible code used by OpenStack does enforce sequence-indentation, something this change does **not**. This change is more relaxed, only request to be consistent inside the file.

So basically this touched only the files that reported inconsistencies, not others.